tests/common/ping/function.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
So - no ping anymore?

Reply:
We dont use ping anymore. Ping tests was taken out when pystein tests were added.